tests/negative/security/tls.go

```diff
@@ -185,6 +175,48 @@ func AppendConfigCustomCertHTTP() types.Test {
 	}
 }
 
+// AppendConfigCABundleCertHTTP appends the ignition configs hosted
+// on the TLS servers using an invalid CA bundle.
+func AppendConfigCABundleCertHTTP() types.Test {
+	name := "tls.config.merge.http.cabundle"
```

Comment:
We're not doing any config merging in this test; we're just fetching files.

Reply:
Do you think this test (https://github.com/coreos/ignition/pull/968/commits/fdc624fa753f65ce17c4356a08abfb183d254975#diff-1ec18742f0f11a112d5e166de55f0e71L153-L154) is also doing the similar thing?

Reply:
That test is similar, and its name `tls.fetchfile.cabundle` accurately claims that it's fetching a file.  For this test, `tls.config.merge.http.cabundle` or `tls.config.fetch.http.cabundle` imply that the test has something to do with config fetching or merging, which it doesn't.

Reply:
oops, it looks like the link that I point you to is not the right one. I was talking about this one: https://github.com/coreos/ignition/blob/master/tests/negative/security/tls.go#L154

Reply:
Yeah, `tls.config.merge.http` and `tls.config.merge.http.invalidheader` should be renamed also.

Reply:
Fixed